File: backend/orderService/src/main/java/sen300/bytedriver/orderService/MenuItem.java
/**
 * @author Cambry Partridge
 * @createdOn 5/10/2024 at 12:05 PM
 * @projectName orderService
 * @packageName sen300.bytedriver.orderService;
 */
package sen300.bytedriver.orderService;

public class MenuItem {
    private String name;
    private double price;
    private int quantity;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}

File: backend/cartService/src/main/kotlin/sen300/bytedriver/cartservice/MenuItem.kt
package sen300.bytedriver.cartservice

import org.springframework.data.annotation.Id
import java.io.Serializable

data class MenuItem(
    @Id
    var id: Int? = null,
    val name: String,
    val price: Float,
    val quantity: Int
) : Serializable {
    constructor(name: String, price: Float, quantity: Int) : this(null, name, price, quantity)
    constructor() : this(null, "", 0.0f, 1)
}